Source/Core/UICommon/UICommon.cpp

```diff
@@ -367,16 +367,15 @@ bool TriggerSTMPowerEvent()
   return true;
 }
 
-#if defined(HAVE_XRANDR) && HAVE_X11
-void EnableScreenSaver(Window win, bool enable)
-#else
+#if !defined(HAVE_XRANDR) || !HAVE_XRANDR
```

Comment:
Any reason you switched the order?

Reply:
The call to `X11Utils::InhibitScreensaver` needed to be gated by the same `HAVE_XRANDR`, so it would essentially have two identical `#ifdef`s right after eachother.

I figured it would make more sense to merge them, especially considering they affect the same variable, and that required switching the order around.

Reply:
Ah, that makes sense. Shouldn't you also be checking for HAVE_X11 though? Or is it always defined and true when HAVE_XRANDR is?

Reply:
IMO, we shouldn't be defining HAVE_XRANDR to 0 in the first place. So removing this line could be cleaner: https://github.com/dolphin-emu/dolphin/blob/master/CMakeLists.txt#L440. Then it could be simplified to just `#if HAVE_XRANDR`.

And yes, HAVE_XRANDR implies HAVE_X11.